website_sale_charge_payment_fee/static/src/js/website_sale_fee.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function () {
+
+    var $pay_button = $('.oe_sale_acquirer_button button');
+    $pay_button.prop('disabled', false);
+
+    // When choosing a payment, update the quotation. Disable the 'Pay
+    // Now' button to avoid being redirected to payment acquirer if the delivery carrier update is
+    // not over.
+    var $payment_method = $("#payment_method");
+    var $oe_cart = $(".oe_cart");
+    var $selected_acquirer_id = $oe_cart.find("input[name='selected_acquirer_id']")
+    var selected_acquirer_id = $selected_acquirer_id.val()
+    $payment_method.find("input[name='acquirer']").click(function (ev) {
+        var payment_id = $(ev.currentTarget).val();
+        // we need to check this because website_sale module clicks on checked acquirer at page load:
```

Comment:
?

Reply:
ok, but I don't get the comment and the commented line: is a TODO then? If so, add 'TODO', if not, remove these lines :wink:

Reply:
@simahawk improved comment: it is not a TODO, it is the reason for the presence of ```if (payment_id != selected_acquirer_id)```. Let me know if it is a bit more clear now